.travis.yml

```diff
@@ -16,3 +18,6 @@ matrix:
       env: "CHECK='rubocop -D'"
     - rvm: 2.1.7
       env: "CHECK='rake commits'"
+    - language: generic
```

Comment:
Can we separate out the job as you did for `puppetdb` ... i.e. with a `stage` like https://github.com/puppetlabs/puppetdb/pull/2706/files#diff-354f30a63fb0907d4ad57269548329e3R143 / results at https://travis-ci.org/puppetlabs/puppetdb/builds/456080661?utm_source=github_status&utm_medium=notification